zh_cn.lang

```diff
@@ -151,198 +151,198 @@ gui.xaero_world_server=世界/服务器
+gui.xaero_box_waypoints_distance_scale=到游戏内路径点的距离的文本显示缩放。 \n \n 旧模组版本中的默认值：\n 深度限制：12 \n 游戏内缩放：2.5 \n 名称文本缩放：0.5 \n 距离文本缩放：0.8
+gui.xaero_box_waypoints_name_scale=游戏内路径点名称的文本显示缩放。 \n \n 旧模组版本中的默认值：\n 深度限制：12 \n 游戏内缩放：2.5 \n 名称文本缩放：0.5 \n 距离文本缩放：0.8
+gui.xaero_box_waypoints_scale=游戏内路径点的缩放。使用它来更改所有游戏内路径点的缩放，包括你附近的路径点。使用“深度限制”设置专门影响远处路径点的大小。 \n \n 旧模组版本中的默认值：\n 深度限制：12 \n 游戏内缩放：2.5 \n 名称文本缩放：0.5 \n 距离文本缩放：0.8
+gui.xaero_waypoint_shared=%1$s 分享了一个名为“%2$s”的路径点！ §2§n[添加]
+gui.xaero_waypoint_shared_dimension=%1$s 分享了一个来自 %3$s 的名为“%2$s”的路径点！ §2§n[添加]
 gui.xaero_waypoint_unknown_dimension=未知维度
-gui.xaero_uses_worldmap_value=该项的值由世界地图模组来设置！
-gui.xaero_uses_worldmap_hard_value=使用世界地图区块时，无法配置设置的值！
-gui.xaero_dots_style=表示实体的点的样式
+gui.xaero_uses_worldmap_value=此设置的值来自世界地图设置！
+gui.xaero_uses_worldmap_hard_value=使用世界地图区块时无法配置此设置的值！
+gui.xaero_dots_style=实体点样式
 gui.xaero_dots_style_default=默认
 gui.xaero_dots_style_legacy=旧版
 gui.xaero_slopes_default_3d=默认 3D
 gui.xaero_slopes_default_2d=默认 2D
 gui.xaero_ui_scale=UI 尺寸
 gui.xaero_ui_scale_auto=自动
 gui.xaero_ui_scale_mc=Minecraft
-gui.xaero_box_ui_scale=文本和其他UI元素的尺寸。\n不适用于放大或调整大小，这两项有单独的选项设置。
+gui.xaero_box_ui_scale=文本和其他小地图UI元素的缩放。 不适用于推近或调整大小。有单独的选项。
 gui.xaero_pushbox_normal=正常
-gui.xaero_pushbox_screen_height=窗口高度
-gui.xaero_pushbox_boss_health=BOSS 血量条碰撞箱类型
-gui.xaero_pushbox_potion_effects=药水效果碰撞箱类型
-gui.xaero_box_pushbox_boss_health=在原版BOSS血量条周围，拥有一个长方形的透明碰撞箱，这会将模组的界面挤开。 \n  \n “正常”模式下碰撞箱会等同于血量条，并仅将接口推动必要的距离。 \n  \n “窗口高度”选项会将碰撞箱拉伸到整个屏幕，并通过“正常”模式的高度垂直移动碰撞箱。 \n 当一列中含有多个窗口并希望所有窗口都能显示时，这将非常有用。
-gui.xaero_box_pushbox_potion_effects=在原版药水状态栏周围，拥有一个长方形的透明碰撞箱，这会将模组的界面挤开。 \n  \n “正常”模式下碰撞箱会等同于血量条，并仅将接口推动必要的距离。 \n  \n “窗口高度”选项会将碰撞箱拉伸到整个屏幕，并通过“正常”模式的高度垂直移动碰撞箱。 \n 当一列中含有多个窗口并希望所有窗口都能显示时，这将非常有用。
+gui.xaero_pushbox_screen_height=屏幕高度
+gui.xaero_pushbox_boss_health=Boss生命条碰撞箱
+gui.xaero_pushbox_potion_effects=药水效果状态栏碰撞箱
+gui.xaero_box_pushbox_boss_health=原版Boss生命条周围有一个隐形碰撞箱将模组的界面推开。 \n \n 正常类型的碰撞箱等于它的大小，并且只将界面推开必要的距离。 \n \n 屏幕高度类型的碰撞箱拉伸到整个屏幕高度，并且总是以正常类型的高度垂直移动碰撞的界面。 当你在一列中有多个界面并希望所有这些界面都被推开时，这将非常有用。
+gui.xaero_box_pushbox_potion_effects=原版药水效果状态栏周围有一个隐形碰撞箱将模组的界面推开。 \n \n 正常类型的碰撞箱等于它的大小，并且只将界面推开必要的距离。 \n \n 屏幕高度类型的碰撞箱拉伸到整个屏幕高度，并且总是以正常类型的高度垂直移动碰撞的界面。 当你在一列中有多个界面并希望所有这些界面都被推开时，这将非常有用。
 gui.xaero_minimap_frame=边框样式
 gui.xaero_minimap_frame_color=边框颜色
-gui.xaero_box_minimap_frame_color=当边框风格不是默认时的边框颜色。
+gui.xaero_box_minimap_frame_color=边框样式不是默认时的小地图边框颜色。
 gui.xaero_minimap_frame_default=默认
 gui.xaero_minimap_frame_colored_thick=彩色
 gui.xaero_minimap_frame_colored_thin=彩色薄边框
 gui.xaero_compass_north=北
 gui.xaero_compass_east=东
 gui.xaero_compass_south=南
 gui.xaero_compass_west=西
-gui.xaero_compass_scale=地图上罗盘文字尺寸
-gui.xaero_box_compass_scale=当罗盘位置在地图上时的文字（东南西北）的尺寸。
+gui.xaero_compass_scale=地图上的指南针缩放
```

Comment:
Make sure this is correct for "Compass Scale On The Map".

Reply:
"compass" is "指南针" in Chinese，But I also noticed that in the game they are actually "text or icons representing directions". So its full name is actually "Compass Text"?

Reply:
I noticed that it translates back as "compass zoom", but should be "size" or "scale". Could still be correct, of course.

Reply:
"缩放" == "Scale", no problem.

I'm actually more confused about this:

> "compass" is "指南针" in Chinese，But I also noticed that in the game they are actually "text or icons representing directions". So its full name is actually "Compass Text"?

because the pictures of the "compass" are (contain pointers, orientation text/icons):
![image](https://user-images.githubusercontent.com/57629614/203377823-e2b936f8-318a-40dd-9c68-9a18b61bcee3.png)

Reply:
> "缩放" == "Scale", no problem.

Great!
> "compass" is "指南针" in Chinese，But I also noticed that in the game they are actually "text or icons representing directions". So its full name is actually "Compass Text"?

I think just saying "compass" should be fine. It's also not ideal in English to call it "compass" instead of "cardinal directions", but that would take too much space. It serves the same purpose as a compass, so I think it can be called that. Just think of it as another type of compass.

Reply:
After discussion with other translators, I decided to translate it as "罗盘".

Reply:
Great!